standard/lexical-structure.md

```diff
@@ -77,7 +77,8 @@ If the following token is among this list, or an identifier in such a context, t
 > *Note*: These rules are not applied when parsing a *type_argument_list* in a *namespace_or_type_name* ([§7.8](basic-concepts.md#78-namespace-and-type-names)). *end note*
 <!-- markdownlint-disable MD028 -->
 
-<!-- markdownlint-enable MD028 -->
+> *Note*: In an *argument_value* of the form `out local_variable_type? variable_reference`, *local_variable_type* may be a generic type; for example: `F(out A<B> name)`. As the language grammar for the argument uses *variable_reference* (which is really *expression*), this context is subject to the disambiguation rule. In this case the closing `>` is followed by an identifier. *end note*
```

Comment:
This is inaccurate regardless of whether we use Nigel's suggestion - the grammar *used* to use *variable_reference* but it doesn't any more.

Reply:
This was also added when I was resolving merge conflicts. It could be reverted back. Let's discuss at the meeting.

Reply:
This xref looks good.

Reply:
Action from 2023-04-19: @BillWagner  to remove the note.